python/lists.md

```diff
@@ -68,6 +68,21 @@ specific indices from the list.
 [4, 2, 9, 45]
 ```
 
+{% callout "Functions and keywords" %}
+`del` is a language `keyword` and not a function. The difference in syntax is
+that functions take their arguments between parentheses:
+
+```python
+my_function(arg1, arg2, arg3...)
+```
+
+whereas builtins take their arguments _without parentheses_:
+
+```python
+del a[-2]
+```
+{% endcallout %}
```

Comment:
We should choose the language to be consistent with here. I use “builtins” to refer to anything that’s available in the global namespace in a clean shell, like `dir`, `True`, and `list`. These aren’t “keywords” like `del` and `import`. (I guess I’d say that `print` changed from a keyword to a builtin in Python 3.)

Reply:
You are right, I should not have used the word "builtin" there. Let me change that.

Reply:
OK I have changed it completely now by simply saying that "del is a keyword representing an action".